File: BeeCrowd/Python/beecrowd_1164.py
soma = 0

n = int(input())

for i in range(n):
    x = int(input())
    for k in range(1, x//2 + 1):
        if (x % k == 0):
            soma += k
    
    print("%d %seh perfeito" %(x, "" if soma == x else "nao ") )
    soma = 0

File: Cpp/beecrowd_1164.cpp
#include <stdio.h>

int main() {

  int n, x, k, soma = 0;

  scanf("%d", &n);

  for (int i = 0; i < n; ++i) {

    scanf("%d", &x);

    for (k = 1; k < x / 2 + 1; ++k)
      if (x % k == 0) {
        soma += k;
      }

    printf("%d %seh perfeito\n", x, soma == x ? "" : "nao ");

    soma = 0;
  }
}
